green_bit_llm/evaluation/evaluate.py

```diff
@@ -21,16 +21,20 @@
 from pathlib import Path
  
 from lm_eval import evaluator
-
+from vllm.model_executor.layers.logits_processor import _apply_logits_processors
+from vllm import LLM, SamplingParams
 import warnings
 
 warnings.filterwarnings('ignore')
 
+
+
 # default value for arguments
 DEFAULT_MODEL_PATH = "GreenBitAI/Qwen-1.5-1.8B-layer-mix-bpw-2.2"
 DEFAULT_SEQLEN = 2048
 DEFAULT_RANDOM_SEED = 0
 DTYPE = torch.half
+DEFAULT_MODEL_BCKEND = ["vllm", "greenbit-engine"]
```

Comment:
The name seems it should be `AVAILABLE_MODEL_BACKENDS`. It could be expected that the first option is the default, or we also declare the default model backend separately.